code/game/objects/structures/transit_tubes/transit_tube_pod.dm

```diff
@@ -165,7 +179,7 @@
 	//  Goal is to balance towards equal pressure, but ensure some gas
 	//  transfer in both directions regardless.
 	// Feel free to rip this out and replace it with something better,
-	//  I don't really know muhch about how gas transfer rates work in
+	//  I don't really know much about how gas transfer rates work in
 	//  SS13.
```

Comment:
I should probably update this lol
Want me to make a pr to your branch for this? It'll be quick and simple